src/modules/dashboard/components/SuggestionCreate/SuggestionCreate.tsx

```diff
@@ -0,0 +1,97 @@
+import React, { useCallback, FC, useRef } from 'react';
+import { FormikHelpers } from 'formik';
+import { defineMessages } from 'react-intl';
+import * as yup from 'yup';
+
+import { COLONY_TOTAL_BALANCE_DOMAIN_ID, ROOT_DOMAIN } from '~constants';
+import { Form, Input } from '~core/Fields';
+import { withDialog } from '~core/Dialog';
+import { OpenDialog } from '~core/Dialog/types';
+import {
+  Domain,
+  useCreateSuggestionMutation,
+  ColonySuggestionsDocument,
+  ColonySuggestionsQueryVariables,
+} from '~data/index';
+import { Address } from '~types/index';
+
+const MSG = defineMessages({
+  inputLabelTitle: {
+    id: 'Dashboard.SuggestionCreate.inputLabelTitle',
+    defaultMessage: 'Suggest features, report bugs, or propose tasks',
+  },
+});
+
+interface FormValues {
+  title: string;
+}
+
+interface InProps {
+  colonyAddress: Address;
+  domainId: Domain['ethDomainId'];
+}
+
+interface Props extends InProps {
+  // Injected via `withDialog`
+  openDialog: OpenDialog;
+}
+
+const validationSchema = yup.object({
+  title: yup.string().required(),
+});
+
+const displayName = 'Dashboard.SuggestionCreate';
+
+const SuggestionCreate = ({ colonyAddress, domainId, openDialog }: Props) => {
+  const inputRef = useRef<HTMLInputElement>(null);
+
+  const [createSuggestion] = useCreateSuggestionMutation();
+
+  const handleSubmit = useCallback(
+    ({ title }: FormValues, { resetForm }: FormikHelpers<FormValues>) => {
+      openDialog('ConfirmDialog')
```

Comment:
So @collinvine mentioned in Asana that we don't need a confirmation dialog.

Pressing "return" should just create the suggestion

See here: 
![Screenshot from 2020-01-21 09-38-36](https://user-images.githubusercontent.com/1193222/72784595-d56d5e00-3c31-11ea-9bbf-fc7eb5f2a03c.png)
https://app.asana.com/0/1156749902660977/progress

So we should either remove it, or take this up with @JoinColony/product

Reply:
Ah, somehow I missed that comment. Thanks for the update! I'm in favor of removing it - we can just submit by pressing `return` 🙂 

I'll remove it in a new PR 👍

Reply:
Update: removed via #1984